test/runtests.jl

```diff
@@ -162,10 +157,10 @@ ds = Data.stream!(f, Data.Table)
 @test ds[1,1].value == 1.0
 @test string(ds[1,2].value) == "2.0"
 @test string(ds[2,2].value) == "NULL"
-f = CSV.Source(dir * "test_missing_value_NULL.csv";null="NULL")
+f = CSV.Source(dir * "test_missing_value_NULL.csv";null=Compat.UTF8String("NULL"))
```

Comment:
Do you also need this one and the two other cases below? It would be nice if `Source` converted its argument to the correct string type.

Other than that, looks good to me.